packages/@sfx/sayt-driver-plugin/test/unit/browser/sayt-driver-plugin.test.ts

```diff
@@ -0,0 +1,193 @@
+import { expect, spy, stub } from '../../utils';
+import { SaytDriverPlugin } from '../../../src/index';
+import { DomEventsPlugin } from '../../../../dom-events-plugin/src/index';
+import { SaytPlugin } from '../../../../sayt-plugin/src';
+
+describe('Sayt Driver Plugin', () => {
+  let Driver: any;
+
+  beforeEach(() => {
+    Driver = new SaytDriverPlugin();
+    Driver.register({
+      'dom_events': new DomEventsPlugin().register({}),
+      'sayt': new SaytPlugin({
+        subdomain: 'macystest',
+        https: true,
+      }).register(),
+    });
+  });
+
+  describe('get metadata()', () => {
+    it('should specify a plugin name of sayt-driver', () => {
+      expect(Driver.metadata.name).to.equal('sayt_driver');
+    });
+
+    it('should have two dependencies: "dom_events" and "sayt"', () => {
+      expect(Driver.metadata.depends).to.deep.equal(['dom_events', 'sayt']);
+    });
+  });
+
+  describe('.register()', () => {
```

Comment:
No need for the `.`. This applies to the others as well.

```suggestion
  describe('register()', () => {
```